### ISSUE TITLE
How to train a new human image to an existing pretrained model when fine-tuning?

### ISSUE THREAD

**Issue by ghzmwhdk777** [ultralytics/ultralytics#2490]
I've looked for issues before, but it seems that they were all new ways to learn your own custom data. Sorry if you have a similar issue.

1. I want to additionally train human images that were not detected by the yolov8 model.
For example, I have 1000 new person photos and I want to train them on an existing person class. This means that class accuracy for person will be maintained, and new person photos will also need to be detected as 'person'. The final output should look like person, dog, cat... Is it possible? 

2. As the second option, I just want to define and train 1000 new person photos as a new class called 'person2'. At this time, is it possible to maintain pretrained class information such as person, dog, and cat? The final output should look like person, person2, dog, cat...

I would appreciate it if you could post an example of both methods.

**Comment by schelmi1** [comment]
I'd like to extend your question instead of opening a new issue.

I have a custom trained model where no coco classes are detected anymore (which is okay)
However i want to add new custom classes to the custom trained model.

How do i preserve the already trained custom classes?

**Comment by glenn-jocher** [comment]
Hello @schelmi1, thank you for your question! 

To add new custom classes to a previously trained YOLOv8 model, you will need to use transfer learning. You can initialize your new model with the weights of your previously trained model, and then continue training the model with your new custom classes.

To do this, you need to update the **number of classes** in your model configuration file to include your new custom classes, and then run the training command on your new data, starting from a high learning rate and gradually decreasing it as the model trains.

It is also recommended to freeze the weights of the initial layers of the model during transfer learning, to avoid overfitting and to save time during training.

We hope this explanation was helpful! If you have any further questions, please don't hesitate to ask.
